site/ar_LB.all.json

```diff
@@ -1338,5 +1338,20 @@
   },
   "shopping_card_update_reason_expired": {
     "other": "قم بتحديث بطاقتك الائتمانية. البطاقات المدعومة هي Visa و Mastercard و American Express."
+  },
+  "shopping_discount_applied_promo": {
+    "other": "تم تطبيق الكود بنجاح."
+  },
+  "shopping_error_promo_only_purchases_must_be_100_percent_discounted": {
```

Comment:
Can we please _not_ have these nightmarish error code concatenated translation keys?  They are a pain to maintain:

The `promo_only_purchases_must_be_100_percent_discounted` string doesn't exist in core-template or Relish so to find the usage I have to search for `shopping_error_` and guess which of the 5 results actually wants to use this translation.

Reply:
It arrives in the attached relish pr that's in review. Would you prefer it to be shorter or follow a different naming scheme?

Reply:
I would prefer the keys to be hard coded in Relish [like this](https://github.com/indiereign/shift72-web/blob/ee6eb5061a8f6a136fb624fb8ceb031fb3e63b74/src/transactional/shopping/shopping.service.js#L10).

Reply:
Have hardcoded the error in the associated relish per https://github.com/indiereign/shift72-web/pull/316/files

Reply:
Nice.  You're gonna need to remove `shopping_error_` from all these keys then right?

Reply:
Done